backend/installGlobalHook.js

```diff
@@ -57,6 +65,33 @@ function installGlobalHook(window: Object) {
           this._listeners[evt].map(fn => fn(data));
         }
       },
+      // Fiber-only:
+      supportsFiber: true,
+      _fiberRoots: {},
+      getFiberRoots(rendererID) {
+        return this._fiberRoots[rendererID] || (this._fiberRoots[rendererID] = new Set());
+      },
+      onCommitFiberUnmount: function(rendererID, fiber) {
+        // TODO: can we use this for roots too?
+        if (this.helpers[rendererID]) {
+          this.helpers[rendererID].handleCommitFiberUnmount(fiber);
+        }
+      },
+      onCommitFiberRoot: function(rendererID, root) {
+        const mountedRoots = this.getFiberRoots(rendererID);
+        const current = root.current;
+        const isKnownRoot = mountedRoots.has(root);
+        const isUnmounting = current.memoizedState == null || current.memoizedState.element == null;
```

Comment:
cc @acdlite This is regard to roots but I don't really have a better idea.

We could possibly make `onCommitFiberRoot` accept the direct children of roots instead of the actual root but it's nice to be able to have the mount container target here too for various debug data.